options/locale/locale_en-US.ini

```diff
@@ -849,18 +849,18 @@ oauth2_application_create_description = OAuth2 applications gives your third-par
 oauth2_application_remove_description = Removing an OAuth2 application will prevent it to access authorized user accounts on this instance. Continue?
 
 authorized_oauth2_applications = Authorized OAuth2 Applications
-authorized_oauth2_applications_description = You've granted access to your personal Gitea account to these third party applications. Please revoke access for applications no longer needed.
+authorized_oauth2_applications_description = You have granted access to your personal Gitea account to these third party applications. Please revoke access for applications you no longer need.
 revoke_key = Revoke
 revoke_oauth2_grant = Revoke Access
 revoke_oauth2_grant_description = Revoking access for this third party application will prevent this application from accessing your data. Are you sure?
-revoke_oauth2_grant_success = You've revoked access successfully.
+revoke_oauth2_grant_success = Access revoked successfully.
```

Comment:
IIRC "revoke" is a "vt": revoke something or something is revoked

Reply:
I'm trying make this string consistent with other messages like `pulls.merged_success`, `pulls.required_status_check_failed`, `pulls.update_branch_success`, `settings.transfer.success`, `settings.archive.success`.

These examples are a bit more detailed, but I found it better than changing the string to something like "You have successfully revoked access to this application." or "This application's access was revoked".

I also thing that `you've` should be removed, I'll make another commit for that.